fhir-server-test/src/test/java/com/ibm/fhir/server/test/ExportOperationTest.java

```diff
@@ -280,26 +293,30 @@ public void testGroup() throws Exception {
         savedGroupId2 = getLocationLogicalId(response);
 
         // Next, call the 'read' API to retrieve the new group and verify it.
-        response = target.path("Group/" + savedGroupId2).request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
+        response      = target.path("Group/" + savedGroupId2).request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
         assertResponse(response, Response.Status.OK.getStatusCode());
         responseGroup = response.readEntity(Group.class);
         assertNotNull(responseGroup);
         assertTrue(responseGroup.getMember().size() == 2);
 
         // (3) Modify the first group to contain the second group and patient1 only, this creates a circle reference situation.
         members.clear();
-        member = Member.builder().entity(Reference.builder()
-                .reference(com.ibm.fhir.model.type.String.of("Patient/" + savedPatientId)).build())
-                .build();
+        member =
+                Member.builder()
+                        .entity(Reference.builder()
+                                .reference(com.ibm.fhir.model.type.String.of("Patient/" + savedPatientId)).build())
+                        .build();
         members.add(member);
-        member = Member.builder().entity(Reference.builder()
-                .reference(com.ibm.fhir.model.type.String.of("Group/" + savedGroupId2)).build())
-                .build();
+        member =
+                Member.builder()
+                        .entity(Reference.builder()
+                                .reference(com.ibm.fhir.model.type.String.of("Group/" + savedGroupId2)).build())
+                        .build();
         members.add(member);
-        group = group.toBuilder().id(savedGroupId).member(members).build();
+        group    = group.toBuilder().id(savedGroupId).member(members).build();
```

Comment:
space

Reply:
fixed, I was playing with save and alignment